learning_resources_search/views.py

```diff
@@ -57,9 +57,11 @@ def get(self, request):
             return Response(SearchResponseSerializer(response).data)
         else:
             errors = {}
-
-            for key, errors_dict in request_data.errors.items():
-                errors[key] = list(set(chain(*errors_dict.values())))
+            for key, errors_obj in request_data.errors.items():
```

Comment:
This change is needed because the error messages for serializers.BooleanField are a list while the errors for serializers.ListField is a dict with the list position of the element that has errors as the key.